forge/test/mlir/test_ops.py

```diff
@@ -1320,3 +1320,73 @@ def forward(self, x):
     co_out = [co.to("cpu") for co in co_out]
     fw_out = [fw_out] if isinstance(fw_out, torch.Tensor) else fw_out
     assert all([compare_with_golden(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
+
+
+@pytest.mark.parametrize("shape", [(1, 3, 32, 32)])
+@pytest.mark.parametrize(
+    "padding",
+    [
+        (1, 1, 1, 1),
+        (1, 1, 2, 2),
+    ],
+)
+@pytest.mark.xfail(reason="'ttnn.conv2d' op Bias must only have data on the final dimenstion")
+def test_padding_with_conv2d(shape, padding):
```

Comment:
Let's rename it to `test_conv2d_with_padding`. Let's focus on the op type first for easier search. And if not already, put this test next to the existing conv2d test :))

Reply:
Updated